web/src/main/java/org/phoenixctms/ctsms/web/jersey/resource/ArgsUriPart.java

```diff
@@ -84,10 +71,6 @@ private Method getAnnotatedMethod(String resource) throws Exception {
 		return arguments;
 	}
 
-	public LinkedHashMap<String, Object> getArgMap(UriInfo uriInfo) throws Exception {
```

Comment:
not sure how to know if they are unused, can eg. be invoked using reflection.

Reply:
where and how should this be picket up by reflection? its in the resource package, but the class doesn't have any jax-rs annotations. can you give me pointers where this might be used?

Reply:
hm, does the non-swagger index (GET http://localhost:8080/ctsms-web/rest) still work, or the indexes of lists&counts (eg. GET http://localhost:8080/ctsms-web/rest/trial/list), and particular rails of them (eg. GET http://localhost:8080/ctsms-web/rest/trial/7284018/list/inquiry)?

Reply:
Yes, I've tested them an all three still work. Are there POST/PUT actions that might used `getArgMap` and co?

Reply:
maybe double check the rails tha call (set|get)(Inquiry|ProbandlIstEntryTag|Ecrf)Values().

i need to double check your changes to the UriArgs in detail